src/amo/components/CollectionAddAddon/index.js

```diff
@@ -131,7 +131,7 @@ export class CollectionAddAddonBase extends React.Component<InternalProps> {
         errorHandlerId: errorHandler.id,
         filters,
         slug: collection.slug,
-        userId: collection.authorUsername,
+        userId: String(collection.authorId),
```

Comment:
Why does this need to be a string? Should that be changed now that we know it will always be an id?